seg_lapa/config_parse/logger_conf.py

```diff
@@ -54,6 +54,8 @@ def get_logger(self, cfg: DictConfig) -> pl_loggers.WandbLogger:
         # The argument names to wandb are different from the attribute names of the class.
         # Pop the offending attributes before passing to init func.
         args_dict = asdict_filtered(self)
+        if args_dict["save_dir"] is None:
+            args_dict["save_dir"] = save_dir
         run_name = args_dict.pop("run_name")
         run_id = args_dict.pop("run_id")
```

Comment:
```suggestion
```
We're overwriting the run_id argument.

At this point, I don't think we need the run_id in config file. Please remove corresponding entry in .yaml config files and it's dataclass.